package-lock.json

```diff
@@ -1,12526 +1,8 @@
 {
   "name": "@triniti/cms",
-  "version": "0.18.4",
-  "lockfileVersion": 2,
+  "version": "0.18.5",
+  "lockfileVersion": 1,
```

Comment:
@meghedi dont want to go backwards on the version

Reply:
@efjacobson @gdbrown  updated, now it uses `"lockfileversion": 3